### ISSUE TITLE
Maybe a error, on the FareAttributes Trasformer

### ISSUE THREAD

**Issue by p4535992** [OpenTransport/gtfs-csv2rdf#16]
Maybe I'm wrong, but this part of the code is correct?
```javascript
if (data["currency_time"]) {
    this.push({ subject: subject, predicate: "http://schema.org/priceCurrency", object : '"' + data["currency_type"] + '"' });
  }
```
I'm not seeing the property "currency_time" on the documentation of gtfs.
If it's correct perhaps we should add a condition, like this:
```javascript
if (data["currency_time"] && data["currency_type"]) {
    this.push({ subject: subject, predicate: "http://schema.org/priceCurrency", object : '"' + data["currency_type"] + '"' });
  }
```

**Comment by pietercolpaert** [comment]
Indeed! That's a typo! Sorry for the late response. Can you pull request the fix?

**Comment by p4535992** [comment]
Sure, i make a pull request on the evening.

P.S. Can you update the homepage url on [http://lov.okfn.org/dataset/lov/vocabs/gtfs](http://lov.okfn.org/dataset/lov/vocabs/gtfs) i'm really interested on that ontology, but i can't find a link to the ontology model.

Greetings.

**Comment by pietercolpaert** [comment]
You mean include a link to http://vocab.gtfs.org?

**Comment by p4535992** [comment]
yes, sorry, i haven't think to search that on github, very stupid me.
Greetings.

2016-07-20 13:39 GMT+02:00 Pieter Colpaert <notifications@github.com>:

**Comment by pietercolpaert** [comment]
No no, you can leave this open! I will do 2 things here:
 * [ ] Make a clear link to http://vocab.gtfs.org from within the ontology
 * [ ] merge your PR
